app/src/main/java/org/connectbot/ConsoleActivity.java

```diff
@@ -308,7 +307,7 @@ public void onCreate(Bundle icicle) {
 		inflater = LayoutInflater.from(this);
 
 		pager = (ViewPager) findViewById(R.id.console_flip);
-		registerForContextMenu(pager);
+		// registerForContextMenu(pager);
```

Comment:
Feel free to totally remove the context menu in this CL if it doesn't make sense with this UX. I think we can figure something out for paste that works nicer with this flow.